docs/reference/search/search.asciidoc

```diff
@@ -428,29 +428,37 @@ Period of time used to extend the life of the PIT.
 <<query-dsl,Query DSL>>.
 
 [[search-api-body-runtime]]
+// tag::runtime-mappings-def[]
 `runtime_mappings`::
-(Optional, object)
-Defines a <<runtime-search-request,runtime field>> in the search request that
-exist only as part of the query. Fields defined in the search request take
-precedence over fields defined with the same name in the index mappings.
+(Optional, object of objects)
+Defines one or more <<runtime-search-request,runtime fields>> in the search
+request. These fields take precedence over mapped fields with the same name.
 +
 .Properties of `runtime_mappings` objects
 [%collapsible%open]
 ====
+
+`<field-name>`::
+(Required, object)
+Configuration for the runtime field. The key is the field name.
++
+.Properties of `<field-name>`
+[%collapsible%open]
+=====
+
```

Comment:
@lockewritesdocs I made some minor changes here to make it clear that you could use this parameter to create multiple runtime fields. I also called out that `runtime_mappings` is an object of objects. I found that a little odd as we typically do an array of objects.

I reuse this definition in the EQL search API ref docs.

Feel free to make suggestions or push changes!